test/e2e/httpproxy/007_client_cert_auth_test.go

```diff
@@ -421,3 +425,12 @@ func optUseCert(cert tls.Certificate) func(*tls.Config) {
 		c.Certificates = append(c.Certificates, cert)
 	}
 }
+
+func isReady(cert *certmanagerv1.Certificate) bool {
```

Comment:
tiny nit, maybe name this `certIsReady` just to make it clear

Reply:
:+1: done